README.md

```diff
@@ -45,13 +46,13 @@ Download
 Dependency to just use `EmailAutoCompleteLayout`
 
 ```groovy
-compile 'com.tasomaniac:emailautocompletelayout:0.1'
+compile 'com.tasomaniac:emailautocompletetextview:0.1'
```

Comment:
I cannot believe that I wrote the dependency name wrong. I immediately fixed it. Thank you. You can revert this to prevent conflict.